java/compaction-status-store/src/test/java/sleeper/compaction/status/testutils/DynamoDBCompactionTaskStatusStoreTestBase.java

```diff
@@ -75,12 +79,28 @@ protected static CompactionTaskStatus finishedTaskWithDefaults() {
                 defaultFinishTime().toEpochMilli()).build();
     }
 
+    protected static CompactionTaskStatus finishedTaskWithDefaultsAndDurationInSecondsNotAWholeNumber() {
+        return startedTaskWithDefaultsBuilder().finished(
+                CompactionTaskFinishedStatus.builder()
+                        .addJobSummary(defaultJobSummaryWithDurationInSecondsNotAWholeNumber()),
+                defaultFinishTimeWithDurationInSecondsNotAWholeNumber().toEpochMilli()).build();
```

Comment:
It might be clearer for this to use the default job summary, since it's the task finish time that matters.